pypreprocess/time_diff.py

```diff
@@ -172,7 +172,7 @@ def plot_tsdiffs(results, use_same_figure=True):
     T = len(results['volume_means'])
     S = results['slice_mean_diff2'].shape[1]
     mean_means = np.mean(results['volume_means'])
-    scaled_slice_diff = results['slice_mean_diff2'] / mean_means
+    scaled_slice_diff = results['slice_mean_diff2'] / mean_means ** 2
```

Comment:
Maybe these divisions (rescalings) should be clarified. It's unclear to me what we're trying to do.